src/components/sidebar/routesView/RouteShow.tsx

```diff
@@ -103,26 +103,21 @@ class RouteShow extends React.Component<IRouteShowProps> {
                             {`${routePath.originFi}-${routePath.destinationFi}`}
                         </div>
                         <div className={s.flexInnerRow}>
-                            <div className={s.routePathDate}>
-                                <div
-                                    className={(isWithinTimeSpan) ?
-                                    classNames(s.flexColumn, s.routePathDate, s.highlight) :
-                                    classNames(s.flexColumn, s.routePathDate)}
-                                >
-                                    {'Alk.pvm:'}
+                            <div
+                                className={(isWithinTimeSpan) ?
+                                classNames(s.routePathDate, s.highlight) :
+                                classNames(s.routePathDate)}
+                            >
+                                <div className={s.flexColumn}>
                                     <ReactMoment
                                         date={routePath.startTime}
                                         format='DD.MM.YYYY'
                                     />
                                 </div>
-                            </div>
-                            <div className={s.routePathDate}>
-                                <div
-                                    className={(isWithinTimeSpan) ?
-                                    classNames(s.flexColumn, s.routePathDate, s.highlight) :
-                                    classNames(s.flexColumn, s.routePathDate)}
-                                >
-                                    {'Voim.ast:'}
+                                <div className={s.flexColumn}>
```

Comment:
Don't use flex column wrapper here. Also remove it from those other 2 places as well.

Reply:
Removed many extra classes